lib/livebook/utils.ex

```diff
@@ -782,7 +782,12 @@ defmodule Livebook.Utils do
     Req.Request.append_request_steps(req,
       connect_options: fn request ->
         uri = URI.parse(request.url)
-        connect_options = mint_connect_options_for_uri(uri)
+
+        connect_options =
```

Comment:
```suggestion
        # We use a step, because the configuration depends on the URL,
        # but we allow any specified :connect_options to take precedence.
        connect_options =
```